src/styles/miscStyles.js

```diff
@@ -186,4 +186,10 @@ export default ({ color, backgroundColor, borderColor, cardColor, dividerColor }
     flexDirection: 'column',
     justifyContent: 'flex-end',
   },
+  emptyStateContainer: {
```

Comment:
Not sure if you did that consciously or you didn't notice, but the previous version was named `empty slate` vs now `empty state`.

Yours is actually more correct. The two popular terms are `blank slate` and `empty state`, so good job ;)